### PR TITLE
Update for Scala.io

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 
 |                         |                                                                    |
 | --------                | --------------------                                               |
-| Time                    | Sept 20th, 14:10 - 18:00                                           |
-| Location                | [Exhibition Halls, Rheged Centre](https://goo.gl/maps/t4ufgYH3GKw) |
+| Time                    | Nov 3rd, 14:00 - 18:00                                             |
+| Location                | [CPE, Lyon](https://goo.gl/maps/jzfE7y5vSbz)                       |
 | Conference registration | **Required**                                                       |
 | Event Registration      | Part of conference registration                                    |
 | Food                    | Conference lunch is before the event                               |
```
Comment (julienrf) [inline README.md]: @ValentinKasas is the above information accurate? Would you add anything useful?

Comment (vil1) [reply README.md]: That's almost it.

@propensive will be able to lead work on [magnolia](https://github.com/propensive/magnolia), [contextual](https://github.com/propensive/contextual), [totalitarian](https://github.com/propensive/totalitarian) and "others"

Comment (vil1) [reply README.md]: Oh and there's also @cvogt willing to lead work on [cbt](https://github.com/cvogt/cbt)

Comment (cvogt) [reply README.md]: if there is interest, yes. Writing a CBT plugin is actually quite fast. 4 hours are plenty of time to get something basic out :).

**Pull request merged**